stable_baselines3/common/policies.py

```diff
@@ -194,6 +194,16 @@ def parameters_to_vector(self) -> np.ndarray:
         """
         return th.nn.utils.parameters_to_vector(self.parameters()).detach().cpu().numpy()
 
+    @abstractmethod
```

Comment:
why not having a default behavior?
this would avoid duplicate code later, no?

Reply:
Yes, you're right. I was unsure whether to add default behaviour or not. I thought it might be safer to leave it as an abstract method, as it would force the person to consider this when implementing a new policy.

But I can add the default behaviour if you prefer?

Reply:
yes, please

Reply:
No problem. I've made that change and removed the default behaviour from the other classes.